nibabel/openers.py

```diff
@@ -17,11 +17,45 @@
 GZIP_MAX_READ_CHUNK = 100 * 1024 * 1024  # 100Mb
 
 
+class BufferedGzipFile(gzip.GzipFile):
+    """GzipFile able to readinto buffer >= 2**32 bytes."""
+    def __init__(self, fileish, mode='rb', compresslevel=9, buffer_size=2**32-1):
+        super(BufferedGzipFile, self).__init__(fileish, mode=mode, compresslevel=compresslevel)
```

Comment:
Looks like `gzip.GzipFile` is [not a new-style class](https://travis-ci.org/nipy/nibabel/jobs/91709435#L1043) in 2.6.

I think this should work:

    gzip.GzipFile.__init__(self, fileish, mode=mode, compresslevel=compresslevel)